### ISSUE TITLE
[UI] Error on navigating to Network -> VNF appliances page

### ISSUE THREAD

**Issue by sureshanaparti** [apache/cloudstack#9257]
[UI] Error below on navigating to Network -> VNF appliances page

`Unable to execute API command listtemplates due to missing parameter templatefilter`

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19
~~~

##### SUMMARY

Noticed error while navigating on UI, to  Network -> VNF appliances page

<img width="674" alt="Error_VNFAppliancesPage " src="https://github.com/apache/cloudstack/assets/12028987/c96133e9-d5c1-4a65-883c-470192f21535">

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~
VNF appliances page should be loaded without any error
~~~

##### ACTUAL RESULTS

~~~
VNF appliances page loads with error - expecting some missing parameter
~~~

**Comment by weizhouapache** [comment]
@sureshanaparti @DaanHoogland 
this should be addressed by #8994

**Comment by DaanHoogland** [comment]
yes, duplicate of #8994, closing
